### PR TITLE
Flush redis cache

### PR DESCRIPTION
**Pull request by Lexy2**
After restoration of the DB we restart the backend container but we don't flush the redis cache, and it stays populated with the old info.

This change resolves this nuisance.

### CODE REVIEW

File: .github/workflows/restore-sanitized-db.yml [hunk 1]
```diff
@@ -100,3 +100,8 @@ jobs:
 
       - name: Restart backend
         run: docker-compose -f docker-compose.ssl.local.yml start backend
+
+      - name: Flush Redis cache
```
Comment (Aivean) [inline .github/workflows/restore-sanitized-db.yml:104]: I think this should go before the `Restart backend`.

**Review by Aivean — COMMENTED** [review]
Please add some information re: how you tested it.

**Comment by Lexy2** [comment]
> Please add some information re: how you tested it.

Ran the command locally on the staging server, setting the environment variable first.

**Review by Aivean — APPROVED** [review]
LGTM

**Pull request merged**